descriptors/ModuleDescriptor-template.json

```
@@ -13,6 +13,14 @@
           "pathPattern": "/circulation-bff/requests/allowed-service-points",
           "permissionsRequired": ["circulation-bff.requests.allowed-service-points.get"],
           "modulePermissions": []
+        },
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/circulation-bff/requests/search-instances",
```

I don't think the search itself has anything to do with requests

```suggestion
          "pathPattern": "/circulation-bff/search-instances",
```